README.md

```diff
@@ -9,3 +9,29 @@ For more information about the control container, including how to use it and ho
 
 You'll need Docker 17.06.2 or later, for multi-stage build support.
 Then run `make`!
+
+## Connecting to AWS Systems Manager (SSM)
+
+Starting from v0.5.0, users have the option to pass in their own activations for SSM. This is for users that want to set up on-premises virtual machines (VMs) in their hybrid environment as managed instances.
+
+Users can add their [own activations](https://docs.aws.amazon.com/systems-manager/latest/userguide/sysman-managed-instance-activation.html) by populating the control container's user-data with a base64-encoded JSON block.
+
+To use hybrid activations for managed instances you will want to generate a JSON-structure like this:
+
+```
+{
+  "ssm": {
```

Comment:
Where would one find this information?  Can we point them to the top level docs for this?